guide/additional-info/hosting.md

```diff
@@ -0,0 +1,68 @@
+# Hosting your bot on a VPS
+
+A bot requires itself to be hosted in order for it to run, however, it would be unideal to host it on your computer all day without shutting it down. That's where a **Virtual Private Server**(VPS) comes into place. A good VPS can mark a turning point for your project as you can host local databases, and it is reliable as well. A VPS suitable for your Discord bot is typically cheap, ranging from around $2-10/month.
+
+## Well known affordable providers
+
+We have gathered a small list of reputable and inexpensive VPS providers that you may rely on for getting your job done:
+* [OVH](https://www.ovh.com/us/vps/) - Starting at $3.35/month for 1 core, 2GB RAM, 20GB SSD
+* [DigitalOcean](https://www.digitalocean.com/) - Starting at $5/month for 1 core, 1GB RAM, 25GB SSD
+* [Linode](https://www.linode.com/) - Starting at $5/month for 1 core, 1GB RAM, 25GB SSD
+* [Vultr](https://www.vultr.com/) - Starting at $2.50/month for 1 core, 512MB RAM, 10GB SSD
+* [AWS Lightsail](https://amazonlightsail.com/) - Starting at $3.50/month (first month free) for 1 core, 512MB RAM, 20GB SSD
+
```

Comment:
I think I'll have to skip on that sorry. Haven't heard of it before and it would be good to keep it minimal to the *more* famous providers

Reply:
I've never heard of GalaxyGate before; what'd different about that?

Reply:
> I think I'll have to skip on that sorry. Haven't heard of it before and it would be good to keep it minimal to the _more_ famous providers

> I decided to add GalaxyGate to the list of VPSs, it offers good amount of resources starting at $3/month. Its used by Dank Memer as wel

This makes entirely no sense, Hetzner is not "famous" enough but GalaxyGate (started in 2018) is? Is the basis of "famous" entirely dependent on whether or not they host DankMemer or other popular Discord bots?